scipy/signal/tests/test_spectral.py

```diff
@@ -1038,6 +1038,7 @@ def test_amplitude(self):
 
     def test_precenter(self):
         # Test if precenter gives the same result as manually precentering.
+        # Precenter is now a non-functional parameter, but the test should still pass.
```

Comment:
I kept this test because there was no reason to remove it. Even though the precenter parameter is now non-functional, the new algorithm should correctly remove any offset added.